lib/web_ui/lib/src/engine/window.dart

```diff
@@ -708,6 +733,21 @@ class ViewConstraints implements ui.ViewConstraints {
       minHeight = size.height,
       maxHeight = size.height;
 
+  factory ViewConstraints.fromJs(JsViewConstraints? constraints, ui.Size? size) {
+    if (size == null) {
+      return const ViewConstraints();
```

Comment:
Hm, I don't think there is a case where `size` is null, since this is the physical measurements of the view, and I don't remember why I added this case (maybe to make this backwards-compatible with our code back in november?), so let's get rid of the nullable `size`.

Good comment!

Reply:
PS: returning null (default) constraints when size is null is the most appropriate, because there's nothing we need to compute. I can't just `return constraints` because the incoming is the `JsConfiguration` version (the one specified by the programmer), and the return type is the "computed" version that the framework needs to work.

If there's no size to be constrained against -> default constraint was probably my thinking.